### DISCUSSION TITLE
network module not  supported

### DISCUSSION THREAD

**Discussion by qrockz** [polybar/polybar#3018] [Build Support]
```
sudo apt install libnl-genl-3-dev
libnl-genl-3-dev is already the newest version (3.5.0-0.1).
```

```
cd build/
cmake ..
```

```
--  Module support:
-- [X]   alsa (1.2.6.1)
-- [ ]   curl
-- [X]   i3
-- [ ]   mpd
-- [ ]   network (wireless-tools)
-- [X]   pulseaudio (15.99.1)
-- [X]   xkeyboard (1.14)
```

But still **no network** support:

I've also installed all the oder dev and optional libs from the wiki. I don't understand why **mpd**, **curl** is also not enabled.
apt install libxcb-xkb-dev libxcb-xrm-dev libxcb-cursor-dev libasound2-dev libpulse-dev i3-wm libjsoncpp-dev libmpdclient-dev libcurl4-openssl-dev libnl-genl-3-dev`

I'm on Linux Mint 21.1 Vera with i3

```
uname -a
Linux t-MS-7C90 5.15.0-58-generic #64-Ubuntu SMP Thu Jan 5 11:43:13 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux
```

Edit:

instead of cmake .. I've tried sudo ./build.sh and from there I could enable all optional features. I think the wiki needs to be updated regarding this.

**Comment by patrick96** [comment]
Most likely what happened is that you did `cmake` before you installed libnl. Since it couldn't find libnl, it fell back to searching for wireless-tools (which it also didn't find).

In such cases, always to a clean build (at the very least delete the `build` folder).
